packages/components/translations/hds/components/application-state/header/en-us.yaml

```
@@ -0,0 +1 @@
+error: ERROR
```

```suggestion
error: ERROR

```
[todo] Looks like the linter is complaining about this file not ending in a new line